### PR TITLE
Move to member function in Matrix class

### PR DESCRIPTION
**Pull request by 200km**
## Overview
Move to member function in Matrix class

## Issue
#333 

## Details
The following functions for Matrix class were moved to the member functions.
- FillUp
- CalcTrace
- Print
- Transpose

The `Unitalize` function was deleted and changed to use MakeIdentityMatrix.

##  Validation results
see CI

## Scope of influence
NA

## Supplement
NA

## Note
Review and Merge after https://github.com/ut-issl/s2e-core/pull/356

We need to change test codes in [this PR](https://github.com/ut-issl/s2e-core/pull/340).

### CODE REVIEW

File: src/library/math/matrix.hpp [hunk 1]
```diff
@@ -46,6 +46,36 @@ class Matrix {
...
+   */
+  T CalcTrace() const;
+
+  /**
+   * @fn Print
+   * @brief Generate all elements to outstream
+   * @param [in] delimiter: Delimiter (Default: tab)
+   * @param [out] stream: Output target(Default: cout)
+   */
+  void Print(char delimiter = '\t', std::ostream& stream = std::cout) const;
```
Comment (suzuki-toshihir0) [inline src/library/math/matrix.hpp:70]: [Q] Vector ( https://github.com/ut-issl/s2e-core/pull/356#discussion_r1139992616 ) と同様で，Printをメンバ関数に入れるかどうか考えたいですね．

**Review by suzuki-toshihir0 — COMMENTED** [review]
https://github.com/ut-issl/s2e-core/pull/356 と同内容ですが，コメントしています．

File: src/library/math/matrix.hpp (hunk 1 above)
Comment (200km) [reply src/library/math/matrix.hpp:70]: Vectorがapproveされるなら、こちらもapproveされるべきだと思うので、よろしくお願いいたします。

**Review by suzuki-toshihir0 — APPROVED** [review]
こちらもApproveします。

**Pull request merged**